centaur/src/test/scala/centaur/s3filechecking/NegTestAwsFileChecker.scala

```diff
@@ -0,0 +1,46 @@
+package centaur.s3filechecking
+
+import java.util
+
+import centaur.test.AwsS3Ops
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.{ObjectListing, S3ObjectSummary}
+import org.scalamock.scalatest.MockFactory
+import org.scalatest.FlatSpec
+import org.specs2.matcher.ShouldMatchers
+
+class NegTestAwsFileChecker extends FlatSpec with ShouldMatchers with MockFactory {
+  val amazonS3Stub = stub[AmazonS3]
+  val awsS3Ops = AwsS3Ops(amazonS3Stub)
+  val testPath = "s3://my-cool-bucket/cromwell/somelogs/myfile"
+  val wrongTestPath = "s3://my-not-so-cool-bucket/somelogs/empty"
+  val objectListing = new ObjectListing
+  val bucketName = "my-cool-bucket"
+  val dirName = "cromwell/somelogs/myfile"
+
+  "AwsFilesChecker" should "return a count of objects at concrete s3 path" in {
+
+    objectListing.setBucketName(bucketName)
+    objectListing.setPrefix(dirName)
+    (amazonS3Stub.listObjects(_: String, _: String)).when(bucketName, dirName).returns(objectListing)
```

Comment:
In the scope of the method that is tested here (`parsePath`) `listObjects` is never called. So you don't need to define this behaviour.